test/Microsoft.AspNet.Mvc.ApiExplorer.Test/DefaultApiDescriptionProviderTest.cs

```diff
@@ -470,13 +470,12 @@ public void GetApiDescription_IncludesResponseFormats()
 
             // Assert
             var description = Assert.Single(descriptions);
-            Assert.Equal(4, description.SupportedResponseFormats.Count);
-
-            var formats = description.SupportedResponseFormats;
-            Assert.Single(formats, f => f.MediaType.ToString() == "text/json");
-            Assert.Single(formats, f => f.MediaType.ToString() == "application/json");
-            Assert.Single(formats, f => f.MediaType.ToString() == "text/xml");
-            Assert.Single(formats, f => f.MediaType.ToString() == "application/xml");
+            Assert.Collection(
+                description.SupportedResponseFormats.OrderBy(f => f.MediaType.ToString()),
```

Comment:
Order isn't in the product code, just ordering here to make the tests less fragile.